### DISCUSSION TITLE
PCAP API method SecurityOnion2

### DISCUSSION THREAD

**Discussion by Xboarder56** [Security-Onion-Solutions/securityonion#3101] [Unsupported Versions] [answered]
Hi all,

I was wondering if there is any formal PCAP pull method for security onion alerts in the new security onion 2 build? I'm trying to build an automated solution to pull the PCAP for the specific alert that I'm investigating into a ticketing system.

**Comment by weslambert** [comment, marked as answer]
There is no supported API at the moment, but this is something we would like to improve in the future.  Thanks for the feedback!
